capstone/src/main/webapp/script.js

```diff
@@ -19,7 +19,7 @@ let precinct = 'SF';
 
 google.charts.load('current', {packages: ['corechart']});
 google.charts.setOnLoadCallback(loadCharts);
-
+//* *zooms and centers map to SF, draws city boarder and creates 3 buttons */
```

Comment:
I think you should be fine with just `// Zooms and centers... and creates 3 buttons` No need for the multi line comment.